src/Compiler/CompilerOptions.cpp

```diff
@@ -212,6 +213,16 @@ static llvm::cl::opt<bool, true> disableKrnlOpFusionOpt(
     llvm::cl::location(disableKrnlOpFusion), llvm::cl::init(false),
     llvm::cl::cat(OnnxMlirCommonOptions));
 
+static llvm::cl::opt<bool, true> disableKrnlBufferReuseOpt(
```

Comment:
NIT: generally "disable" is for a function that is default on, "enable" is for one that is default off. I think you want "enable" here.

Reply:
fixed